Symbology/web/pyBRAT/py_lu.txt

```diff
@@ -1,5 +1,5 @@
 # QGIS Generated Color Map Export File
-INTERPOLATION:DISCRETE
+INTERPOLATION:EXACT
 0,0,97,0,255,Not,Developed
 0.33,164,196,0,255,Agriculture:,Lower,Intensity
```

Comment:
What's with all the commas in the label. did you intend that?

```
Not,Developed
Agriculture:,Lower,Intensity
```

wouldn't  this be better:

```
```
Not Developed
Agriculture: Lower Intensity
```

Reply:
@MattReimer, I've just committed those changes. I'm note sure why those commas were there; I just ended up editing the existing file. Also, how did you create the categorized symbology in QGIS?